collector-bitmovin-player/build.gradle

```diff
@@ -35,8 +35,8 @@ dependencies {
     androidTestImplementation 'com.android.support.test:runner:1.0.1'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
 
-    api 'com.bitmovin.analytics:collector:1.4.2'
-    api 'com.bitmovin.player:playercore:2.+'
+    api 'com.bitmovin.analytics:collector:1.4.3'
+    implementation 'com.bitmovin.player:playercore:2.20.1'
```

Comment:
I don't think this is good - as it will break anyone running 2.21 right?

@matamegger @hawk23 any idea how we can allow a user to select a newer version of bitmovin player (without being locked to this specific version) without updating his version to latest?
The issue with `2.+` was that it would automatically select the latest version all the time

Reply:
Let's discuss this offline, I think I don't fully understand the question.

Reply:
As discussed offline with @rolandgriesser that should not be a problem.